spec/unit/file_system_spec.rb

```diff
@@ -134,6 +134,36 @@ def increment_counter_in_multiple_processes(file, num_procs, options)
     end
   end
 
+  context "read should allow an encoding to be specified" do
+    # First line of Rune version of Rune poem at http://www.columbia.edu/~fdc/utf8/
+    # characters chosen since they will not parse on Windows with codepage 437 or 1252
+    let (:rune_utf8) { "\u16A0\u16C7\u16BB" } # 'ᚠᛇᚻ'
```

Comment:
This assumes that ruby will create a utf8 literal string, but I think that depends how ruby is executed (and potentially the magic encoding comment at the top of this file). It might be safer to just say `{  "\u16A0\u16C7\u16BB".encode('utf-8') }` so you always know what you have, and then remove the `before :each` below.